tests/contrib/io/azure/test_csv_blob.py

```diff
@@ -62,6 +74,103 @@ def make_data_set(load_args=None, save_args=None):
     return make_data_set
 
 
+@pytest.fixture
+def versioned_blob_csv_data_set(load_version, save_version):
+    return CSVBlobDataSet(
+        filepath=TEST_FILE_NAME,
+        container_name=TEST_CONTAINER_NAME,
+        credentials=TEST_CREDENTIALS,
+        blob_to_text_args={"to_extra": 42},
+        blob_from_text_args={"from_extra": 42},
```

Comment:
Minor:  42 is a random number, isn't it? I would use different values for `to_extra` and `from_extra` when testing.

Reply:
Okay, makes sense. Should I change the existing tests as well (they also use the same number).

Reply:
I've changed the ones I added (in `versioned_csv_blob_data_set` but not the ones already existing, let me know if I should change those too.